src/senaite/core/profiles/default/workflows/senaite_sample_workflow/definition.xml

```diff
@@ -1580,6 +1590,16 @@
     </guard>
   </transition>
 
+  <!-- Transition: Multi Results -->
+  <transition transition_id="listing_multi_results" title="Multi Results" new_state="" trigger="USER"
+              allowed="False" before_script="" after_script="" i18n:attributes="title">
+    <action url="" category="workflow" icon="">Multi Results</action>
+    <guard>
+      <guard-expression>python:here.guard_handler("restore")</guard-expression>
+      <guard-permission>senaite.core: Transition: Restore Sample</guard-permission>
```

Comment:
Misspelling: "senaite.core: Transition: Restore Sample" --> "senaite.core: Transition: Multi Results"

Reply:
Thanks for your thorough review and finding all the things I have overlooked! ❤️